style.css

```diff
@@ -0,0 +1,375 @@
+  list-style: none;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+}
+
+.skills li {
+  list-style: none;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+}
+
+.tags li {
+  width: 25px;
+  height: 20px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.languages li:first-child {
+  color: #9490ff;
+  margin-bottom: 10px;
+  height: 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+}
+
+.frameworks li:first-child {
+  color: #9490ff;
+  margin-bottom: 10px;
+  width: 107px;
+  height: 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+}
+
+.skills li:first-child {
+  color: #9490ff;
+  margin-bottom: 10px;
+  width: 107px;
+  height: 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+}
+
+.portfolio {
+  display: none;
+}
+
+.about {
+  display: none;
+}
+
+.contact {
+  display: none;
+}
+
+.mail {
+  display: none;
+}
+
+.icon-bar ul {
+  display: flex;
+  justify-content: center;
+  width: 325px;
+  height: 40px;
+  list-style: none;
+  margin-top: 40px;
+  margin-bottom: 20px;
+  gap: 40px;
+  padding-left: 24px;
+}
+
+.footer {
+  text-align: center;
+  margin-top: 50px;
+  margin-bottom: 70px;
+}
+
+.fa-arrow-down {
+  color: #a2a3ae;
+  font-weight: 500;
+  font-size: 20px;
+  letter-spacing: 2px;
+}
+
+.recent-works h1 {
+  height: 104px;
+  font-weight: 500;
+  font-size: 40px;
+  font-style: normal;
+  color: #9490ff;
+  line-height: 130%;
+  display: flex;
+  align-items: center;
+  padding-left: 24px;
+}
+
+hr {
+  position: absolute;
+  top: calc(50% - 0px / 2 + 0.5px);
+  border: 1px solid #646369;
+}
+
+.featured-image {
+  display: grid;
+  grid-column-start: auto;
+  padding: 24px;
+  gap: 24px;
+  height: 729px;
+}
+
+.image {
+  width: 327px;
+  height: 236px;
+  background-image: linear-gradient(0deg, #f9f9f9, #f9f9f9);
+  align-items: center;
+}
+
+.right-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 24px;
+  height: 469px;
+}
+
+.right-block h2 {
+  height: 105px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.right-block p {
+  width: 327px;
+  height: 204px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #dcdce8;
+}
+
+.tags {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  align-items: center;
+  padding: 10px 12px;
+  width: 325px;
+  height: 40px;
+  color: #fff;
+  list-style: none;
+}
+
+.button-frame {
```

Comment:
- [ ] I kindly suggest that you follow the button styling for your template on Active, Hover and Pressed:

![Screenshot from 2022-05-27 19-07-25](https://user-images.githubusercontent.com/52934128/170766692-760b42c1-2f5b-4ffe-af72-9762dbfccd1b.png)

| Your Design  | Template Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-27 19-06-01](https://user-images.githubusercontent.com/52934128/170766520-a5c36ae2-72d6-4d44-bddb-a55565e1378b.png)| ![image](https://user-images.githubusercontent.com/52934128/170766028-fe0940b2-9ed9-43ac-8316-17d038613cb5.png) |